tools/cleanup.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec opam admin filter --dry-run --resolve=xs-toolstack --resolve upstream-extra-dummy --resolve depext.transition --resolve depext.1.0.5 --or --with-test
+exec opam admin filter --dry-run --resolve=xs-toolstack --resolve upstream-extra-dummy --resolve depext.transition --resolve opam-ed.0.1 --or --with-test
```

Comment:
What does this script do?

Reply:
We should document it.

Reply:
It detects packages that are not referenced.